datascience/tables.py

```diff
@@ -674,8 +681,9 @@ def relabel(self, column_label, new_label):
 
     def copy(self):
         """Return a copy of a Table."""
+        # TODO(denero) Shallow copy by default with an option for deep copy
```

Comment:
Yes, I believe it deep copies because of the np.copy(self[label]), but I
don't think we really need to make deep copies ever. That seems like an
issue for another pull request, though.

On Thu, Jan 28, 2016 at 11:40 PM, Sam Lau <notifications@github.com> wrote:

> In datascience/tables.py
> <https://github.com/data-8/datascience/pull/187#discussion_r51233122>:
>
> > @@ -674,8 +681,9 @@ def relabel(self, column_label, new_label):
> >
> >      def copy(self):
> >          """Return a copy of a Table."""
> > +        # TODO(denero) Shallow copy by default with an option for deep copy
>
> Does this TODO mean that it currently deep copies?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/data-8/datascience/pull/187/files#r51233122>.
>